src/server/wtr-config.js

```diff
@@ -1,11 +1,13 @@
 import { defaultReporter } from '@web/test-runner';
+import { env } from 'node:process';
 import { headedMode } from './headed-mode-plugin.js';
 import { playwrightLauncher } from '@web/test-runner-playwright';
 import { reporter as testReportingReporter } from 'd2l-test-reporting/reporters/web-test-runner.js';
 import { visualDiff } from './visual-diff-plugin.js';
 import { visualDiffReporter } from './visual-diff-reporter.js';
 
 const DEFAULT_PATTERN = type => `./test/**/*.${type}.js`;
+const DEFAULT_TEST_REPORTING = !!env['GITHUB_ACTIONS'];
```

Comment:
I had changed this to `GITHUB_ACTIONS` but wonderig if I should leave it as `CI` since this is more generic. Thoughts?